samples/nrf9160/lwm2m_client/fota.rst

```diff
@@ -0,0 +1,295 @@
+   #. Open the :file:`src/prj.conf` file.
+   #. Change :kconfig:option:`CONFIG_MCUBOOT_IMAGE_VERSION` to ``1.0.1`` and rebuild the sample.
+   #. Update the application and modem firmware by using the :file:`/scripts/fota.py` script:
+
+      .. code-block:: console
+
+         ./scripts/fota.py -id urn:imei:351358811331351 update 0 file app_update.bin update 1 file mfw_nrf9160_update_from_1.3.4_to_1.3.4-FOTA-TEST.bin
+
+   #. Following is an example output of the command:
+
+      .. code-block:: console
+
+         [INFO] fota.py - Client identity: urn:imei:351358811331351
+         [INFO] fota.py - Binary app_update.bin, Size 336032 (bytes)
+         [INFO] coiote.py - Creating fota resource for binary app_update.bin with id lwm2m_client_fota_instance_0
+         [INFO] fota.py - Init setup for instance 0 firmware Update resource:lwm2m_client_fota_instance_0
+         [INFO] fota.py - Binary mfw_nrf9160_update_from_1.3.4_to_1.3.4-FOTA-TEST.bin, Size 14472 (bytes)
+         [INFO] coiote.py - Creating fota resource for binary mfw_nrf9160_update_from_1.3.4_to_1.3.4-FOTA-TEST.bin with id lwm2m_client_fota_instance_1
+         [INFO] fota.py - Init setup for instance 1 firmware Update resource:lwm2m_client_fota_instance_1
+         [INFO] fota.py - Start Firmware Update
+         [INFO] fota.py - Delete Observation Advanced Firmware Update
+         [WARNING] coiote.py - Coiote: Path Advanced Firmware Update was not observed
+         [INFO] fota.py - Write Fota Download url to Advanced Firmware Update.0.Package URI
+         [INFO] fota.py - Write Fota Download url to Advanced Firmware Update.1.Package URI
+         [INFO] coiote.py - Device is Queuemode Coiote have to wait next Registration Update
+         [INFO] fota.py - Download Url Write done
+         [INFO] fota.py - Enable Observation Advanced Firmware Update
+         [INFO] fota.py - Downloading instance: 0
+         [INFO] fota.py - Downloading instance: 1
+         [INFO] fota.py - Download ready instance: 0
+         [INFO] fota.py - Download ready instance: 1
+         [INFO] coiote.py - Device is Queuemode Coiote have to wait next Registration Update
+         [INFO] fota.py - Update started instance: 0
+         [INFO] fota.py - Update started instance: 1
+         [INFO] fota.py - Firmware Update Successfully instance: 0
+         [INFO] fota.py - From:1.0.0-0 to 1.0.1-0
+         [INFO] fota.py - Firmware Update Successfully instance: 1
+         [INFO] fota.py - From:mfw_nrf9160_1.3.4 to mfw_nrf9160_1.3.4-FOTA-TEST
+         [INFO] fota.py - Firmware update process finished
+         [INFO] fota.py - Delete Observation Advanced Firmware Update
+
+   #. Update the modem firmware back to the original released version:
+
+      .. code-block:: console
+
+         ./scripts/fota.py -id urn:imei:351358811331351 update 1 file mfw_nrf9160_update_from_1.3.4-FOTA-TEST_to_1.3.4.bin
+
+   #. Following is an example output of the command:
```

Comment:
This is not a step or a user action and hence can be part of the previous step.

Reply:
I can fix this also